### ISSUE TITLE
Checkbox value <> aria-checked sync issue

### ISSUE THREAD

**Issue by jole78** [reakit/reakit#465]
## 🐛 Checkbox value <> aria-checked sync issue
Don't know this is a bug or something I've missed but thought I'd report it anyway.
When checkbox checked is set from outside the onChange handler aria-checked is not in sync any more.

### Current behavior

useCheckbox value is not in sync with aria-checked unless the checkbox is clicked.

### Steps to reproduce the bug
I saw this in an experiment so I just have the sandbox experiment, hope that's ok.
https://codesandbox.io/s/hungry-glade-74cuk

1. Click the "TOGGLE IT" button and see the props (aria-checked is not synced)
2. Click the CheckBox and it toggles/syncs

### Expected behavior

checked should be in sync with aria-checked

### Possible solutions

If possible, describe how this bug could be fixed.

### Environment
see sandbox
